### DISCUSSION TITLE
Click to open URL now defaults to Google Chrome

### DISCUSSION THREAD

**Discussion by jehoshua7** [keepassxreboot/keepassxc#10801] [Q&A] [answered]
Running 2.7.4 on Kubuntu 24.01

Have always just clicked the URL with the mouse and Firefox opens that URL. Installed Google Chromium yesterday, and now clicking the URL opens Google Chrome. It is not my default browser, Firefox still is. This is not browser integration, just click the link and open.

Firefox is installed via SNAP, could that be the problem ? Chrome was installed by downloading the .DEB and installing. In the past I have uninstalled SNAP/Firefox as it was bloatware, seems to be it still is, so I can do that if necessary ??  Then install Firefox from a PPA

**Comment by jehoshua7** [comment]
I can click a URL from Claws Mail and it uses Firefox, so it does seem a KeepassXC specific issue.

**Reply by phoerious** [reply]
This is not a KeePassXC issue, this is an issue with your Qt configuration, which we cannot really help you with. Claws Mail is a GTK application.

**Reply by jehoshua7** [reply]
The comment about Claws Mail was only an example. The issue/problem is clicking the URL from KeepassXC. It used to open Firefox, now it opens Chrome, despite the fact that Firefox is the default browser.

**Reply by phoerious** [reply, marked as answer]
Still a problem with Qt on your system.

**Reply by jehoshua7** [reply]
> Still a problem with Qt on your system.

Okay thanks for your help. I guess I'll have to ask at https://forum.qt.io/
